### DISCUSSION TITLE
What train to convert Japan VW Golf to EU?

### DISCUSSION THREAD

**Discussion by vvv104** [Mr-MIBonk/M.I.B._More-Incredible-Bash#456] [Q&A]
Hi, I have imported my Golf from Japan to Cyprus. I successfully installed M.I.B to my unit. My current train is MU1201-MHI2_JP_VW48x_P1201. Now I am looking for the FW to convert it to EU. Where can I get the right one? I can provide logs as well as full backup of my current SW made by M.I.B. I found a 28Gb EU map update, but I am not sure this will go without changing to EU completely. Although I fixed FECs in MIB. Thank you for help!

**Comment by mariosmax** [comment]
@vvv104 did you find a solution to this?
